wiki/Guides/Getting_Your_Map_Modded/en.md

```diff
@@ -1,38 +1,43 @@
+- **Hitsound check:** The modder will check your beatmap to make sure that your hitsounding represents the song correctly, is consistent, and is up to ranking standards.
 
-- Typing the `!request` command into chat will make BanchoBot PM you a random map link.
-- Typing the `!request list` command into chat will make BanchoBot PM you a list of the 5 most recent requests plus a link to more.
-- Visiting a [special map listing](https://old.ppy.sh/p/beatmaplist&r=8) on the main site will list the 20 most recent requests plus a few extras.
+## Modding queues
 
-If you post more than two map links in a short period of time, Bancho will ignore them.
+Posting your beatmap in modding queues is the most effective and most widely used way of finding people to mod your beatmaps. Modding queues are threads posted in the [Modding Queues forum](https://osu.ppy.sh/community/forums/60) which allow the creator of the thread to find beatmaps to mod, and for others to easily find modders to mod their beatmaps. Using the forum to find mods is straightforward and effective—just follow the steps below to get started:
 
-It's not the most effective method, but it doesn't hurt to try (just don't count on it).
+1. Go to the [Modding Queues forum](https://osu.ppy.sh/community/forums/60).
+2. Browse through the list of queues, and find one that accepts beatmaps from the game mode which you have mapped for, and make sure it is marked as "open" in the title or opening post.
+3. Read the queue's rules. If you or your beatmap does not meet the requirements listed there, then unfortunately you will have to find another queue to post in.
+4. Read the modder's preferences listed in the queue (if any), as it may save both you and the creator of the queue a lot of time.
+5. Click the `go to last post` button at the bottom of the screen. This is represented by a double right chevron icon.
+6. Type what sort of request you would like, along with a link to your beatmap and any other information the modder has asked you to provide, into the `New reply` box.
+7. Click `Post` when you are finished.
 
-## Modding queues
+It is generally acceptable to post in a few modding queues at a time, so feel free to repeat this process if necessary, or if you want to boost your chances of receiving a mod. Keep in mind, however, that just because you have posted in a modding queue, it does not mean the modder is obligated to mod your beatmap; they will mod it if they want to.
+
+It is worth noting that you should avoid posting in Beatmap Nominators' queues until your mapping is at a near rankable level (unless they have specified that they are looking for novice mappers), as they will most likely ignore your requests otherwise. You are much better off finding normal modders to mod your beatmaps when you are still learning how to create beatmaps.
```

Comment:
```suggestion
It is worth noting that you should avoid posting in Beatmap Nominators' queues until your mapping is at a near-rankable level (unless they have specified that they are looking for novice mappers), as they will most likely ignore your requests otherwise. You are much better off finding normal modders to mod your beatmaps when you are still learning how to create beatmaps.
```